pyproject.toml

```diff
@@ -19,16 +19,15 @@ keywords = [
 license = {text = "Apache-2.0 License"}
 dependencies = [
   "datasets~=2.3",
-  "numpy~=1.22",
-  "tokenizers~=0.12",
   "torch>=1.11",
   "zstandard~=0.17",
-  "transformers==4.20.0",
   "lxml~=4.9",
   "sacremoses~=0.0.53",
   "optuna~=2.10",
   "libarchive-c~=4.0",
   "scikit-learn~=1.1",
+  "tokenizers==0.15",
+  "transformers>=4.20.0",
```

Comment:
To keep the changes minimal, can you test if the following version specifiers work also?
```
transformers==4.37.1
tokenizers~=0.15.0
numpy=~1.26.3
```